<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>find_color_block</class>
 <widget class="QWidget" name="find_color_block">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>490</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>颜色选择</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QCheckBox" name="checkBox_Red">
       <property name="font">
        <font>
         <family>3ds</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>red</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Blue">
       <property name="font">
        <font>
         <family>3ds</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>blue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Yellow">
       <property name="font">
        <font>
         <family>3ds</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>yellow</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>610</y>
     <width>621</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>HSV调整</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>631</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>3ds</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>S 最大值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="S_Slider_max">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>601</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>3ds</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>HSV 范围</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="TextEdit_hsv">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>3ds</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>3ds</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>V 最大值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="V_Slider_max">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>3ds</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>S 最小值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="S_Slider_min">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>3ds</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>H 最小值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="H_Slider_min">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>3ds</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>H 最大值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="H_Slider_max">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>3ds</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>V 最小值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="V_Slider_min">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_img_gray">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>611</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>机械臂</string>
   </property>
   <widget class="QPushButton" name="Button_arm_start">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始工作</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>3ds</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>红色放置</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextBrowser" name="show_red_place"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>3ds</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>蓝色放置</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextBrowser" name="show_blue_place"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>3ds</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>黄色放置</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextBrowser" name="yellow_blue_place"/>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>3ds</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>调整</string>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>171</width>
       <height>161</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>X+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Z+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Z-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>X-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>41</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Y+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>41</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Y-</string>
      </property>
     </widget>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>3ds</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>单位调整数值</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>490</y>
     <width>441</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>识别结果</string>
   </property>
   <widget class="QTextBrowser" name="show_recong_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
